events/imperia_discord_break.txt

```diff
@@ -0,0 +1,2849 @@
+				}
+				random_list = {
+					1 = {
+						trigger = {
+							var:first_option != 206
+							var:second_option != 206
+						}
+						set_variable = {
+							name = third_option
+							value = 206
+						}
+					}
+					1 = {
+						trigger = {
+							var:first_option != 207
+							var:second_option != 207
+						}
+						set_variable = {
+							name = third_option
+							value = 207
+						}
+					}
+					1 = {
+						trigger = {
+							var:first_option != 208
+							var:second_option != 208
+						}
+						set_variable = {
+							name = third_option
+							value = 208
+						}
+					}
+					1 = {
+						trigger = {
+							var:first_option != 209
+							var:second_option != 209
+							exists = scope:discriminated_culture
+						}
+						set_variable = {
+							name = third_option
+							value = 209
+						}
+					}
+					1 = {
+						trigger = {
+							var:first_option != 2010
+							var:second_option != 2010
+						}
+						set_variable = {
+							name = third_option
+							value = 2010
+						}
+					}
+					1 = {
+						trigger = {
+							var:first_option != 2011
+							var:second_option != 2011
+						}
+						set_variable = {
+							name = third_option
+							value = 2011
+						}
+					}
+				}
+			}
+			# 275 - 375
+			0 = {
+				modifier = {
+					add = discord_break_lvl_3_lower
+				}
+				random_list = {
+					# This chooses a DIFFERENT second option
+					1 = {
+						trigger = {
+							var:first_option != 301
+							var:second_option != 301
+						}
+						set_variable = {
+							name = third_option
+							value = 301
+						}
+					}
+					1 = {
+						trigger = {
+							var:first_option != 302
+							var:second_option != 302
+						}
+						set_variable = {
+							name = third_option
+							value = 302
+						}
+					}
+					1 = {
+						trigger = {
+							var:first_option != 303
+							var:second_option != 303
+						}
+						set_variable = {
+							name = third_option
+							value = 303
+						}
+					}
+					1 = {
+						trigger = {
+							var:first_option != 304
+							var:second_option != 304
+						}
+						set_variable = {
+							name = third_option
+							value = 304
+						}
+					}
+					1 = {
+						trigger = {
+							var:first_option != 305
+							var:second_option != 305
+						}
+						set_variable = {
+							name = third_option
+							value = 305
+						}
+					}
+					1 = {
+						trigger = {
+							var:first_option != 306
+							var:second_option != 306
+						}
+						set_variable = {
+							name = third_option
+							value = 306
+						}
+					}
+					1 = {
+						trigger = {
+							var:first_option != 307
+							var:second_option != 307
+						}
+						set_variable = {
+							name = third_option
+							value = 307
+						}
+					}
+				}
+			}
+			# 325 - 400
+			0 = {
+				modifier = {
+					add = discord_break_lvl_3_upper
+				}
+				random_list = {
+					1 = {
+						trigger = {
+							var:first_option != 308
+							var:second_option != 308
+						}
```

Comment:
What? why do we need to check this?

Reply:
Because we dont want the same option appearing twice in the events.
While the second option checks for the presence of the first option (option 3 is not generated yet there), it cannot be shortened down to a comparison between 2 and 3 here, as that would cause option 1 and 3 to collide.
If that was your concern :)